src/lrf.rs

```diff
@@ -1458,11 +1402,9 @@ impl RestorationState {
                 continue;
               }
 
-              sgrproj_stripe_filter(
-                set,
-                xqd,
-                fi,
+              setup_integral_image(
                 &mut stripe_filter_buffer,
+                STRIPE_IMAGE_STRIDE,
```

Comment:
it seems constant over all the function calls, would make sense to embed it inside the function?

Reply:
It's not.  You missed the calls coming in from rdo.rs